lib/vagrant-service-manager.rb

```diff
@@ -8,7 +8,7 @@
 require 'vagrant-service-manager/command'
 require 'vagrant-service-manager/os'
 
-module Vagrant
+module VagrantPlugins
```

Comment:
To use one instance of [`PluginLogger` file](https://github.com/budhrg/vagrant-service-manager/blob/3de6cf2ba92980b0b22b9447c32cf4f5ad2d3392/lib/vagrant-service-manager/plugin_logger.rb)